scripts/build_and_install.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
This seems to be rather close to what `build-gee.sh && install-gee.sh` under `docker/image-definition/scripts` do. We could consider reducing code duplication.